### PR TITLE
Migrate to google/uuid

### PR DESCRIPTION
**Pull request by dsolonenko**
To fix https://github.com/epsagon/epsagon-go/issues/13

### CODE REVIEW

**Comment by enoodle** [comment]
this fails for golang tip, I am looking on how to solve this as I don't think it is very specific to this code.

File: epsagon/tracer.go [hunk 1]
```diff
@@ -77,10 +77,14 @@ func (tracer *epsagonTracer) sendTraces() {
 
 	resp, err := client.Post(tracer.Config.CollectorURL, "application/json", tracesReader)
 	if err != nil {
-		var respBody []byte
-		resp.Body.Read(respBody)
-		resp.Body.Close()
-		log.Printf("Error while sending traces \n%v\n%+v\n", err, respBody)
+		if resp != nil {
```
Comment (enoodle) [inline epsagon/tracer.go]: This is a good change but not part of migrating the uuid library, can you split it to another commit at least?

Comment (dsolonenko) [reply epsagon/tracer.go]: agree, let me move it into another pr. I had to do it because the tests were failing for me.

**Comment by dsolonenko** [comment]
Seems that the build against `tip` is just unstable. I can see from other repositories that people tend to pick specific supported versions, eg https://github.com/olivere/elastic/blob/release-branch.v6/.travis.yml to resolve that :)
